applications/decentralized-pallet-staking-audit-tool.md

```diff
@@ -0,0 +1,374 @@
+# Decentralized pallet-staking Audit Too
+- **Team Name:** Mark Van de Vyver [PhD](https://www.student.uwa.edu.au/course/award-verification-service?family=vyver&family_partial=on&given=mark&given_partial=on&search=Search)
+- **Payment Address:** 15WoVugRLfBAogS1FBmhzxVv8WZqZK1z4KKHALijTZfdVcCm (USDT)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 3
+
+## Project Overview :page_facing_up:
+
+### Overview
+
+A decentralized tool (Excel/OpenOffice workbook) to allow developers, journalists/bloggers, business analysts, regulators and users to confirm the proper functioning of a program of
```

Comment:
This information seems already available pretty easily for me -  for user accounts, they can use the  [Staking Rewards Collector](https://github.com/w3f/staking-rewards-collector)  which makes a CSV file.  Or systemically, you can get the data from [Subscan](https://polkadot.subscan.io/event?address=&module=staking&event=erapaid&startDate=&endDate=&startBlock=&endBlock=&timeType=date&version=9430). At this point it's a pretty simple calculation to calculate the rest of the parameters.